src/Microsoft.Extensions.Logging.AzureWebApps/Internal/BackgroundSink.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Concurrent;
+using System.Threading;
+using Serilog.Core;
+using Serilog.Events;
+
+// TODO: Might want to consider using https://github.com/jezzsantos/Serilog.Sinks.Async 
+// instead of this, once that supports netstandard
+namespace Microsoft.Extensions.Logging.AzureWebApps.Internal
+{
+    /// <summary>
+    /// A background sink for Serilog
+    /// </summary>
+    public class BackgroundSink : ILogEventSink
```

Comment:
Serilog logger has been available for a while and is probably used for web apps frequently, so wondering...do they have not have this kind of support already built in? (like I am a bit surprised if they go to the local disk for every write)